src/main/java/org/broadinstitute/consent/http/util/DarUtil.java

```diff
@@ -22,6 +23,14 @@
         return Collections.emptyList();
     }
 
+    public static boolean hasUserRole(User user, Integer roleId) {
+        if (Objects.isNull(user) || Objects.isNull(user.getRoles())) {
+            return false;
+        } else {
+            return user.getRoles().stream().anyMatch((role) -> role.getRoleId().equals(roleId));
+        }
+    }
```

Comment:
nit: `DarUtils` doesn't seem like the right home for a method that checks on user roles. This cold be a method on the `User` model itself if you really need it.